interface-definitions/interfaces_utunnel.xml.in

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="interfaces">
+    <children>
+      <tagNode name="utunnel" owner="${vyos_conf_scripts_dir}/interfaces_custom_tunnel.py">
```

Comment:
Can you share exemple of CLI ?
I didn't get how does It looks like. Part of config you do from CLI, another part from some hardcoded path from .yml